llvm/lib/Target/EVM/EVMFinalizeStackFrames.cpp

```diff
@@ -171,10 +171,14 @@ bool EVMFinalizeStackFrames::runOnModule(Module &M) {
 
   // Check if it is valid to replace frame indices.
   if (TotalStackSize > 0) {
-    if (TotalStackSize > StackRegionSize)
+    if (TotalStackSize > StackRegionSize) {
+      // If the front-end has registered the handler, the following function
+      // should not return. Otherwise, it has no effect.
+      report_evm_stack_error(TotalStackSize);
```

Comment:
It's desirable to make default `report_evm_stack_error` behavior equivalent to `report_fatal_error` below and remove that line.

Reply:
Yes. Something alike.